File: packages/vscode/src/onboarding.ts
export const CLI_INSTALL_HINT =
  "Install the Grok Build CLI: curl -fsSL https://x.ai/cli/install.sh | bash && grok login"

export const missingCliMessage = (searched: ReadonlyArray<string>): string =>
  `Grok CLI not found. ${CLI_INSTALL_HINT}\nLooked in: ${searched.join(", ")}`

File: core/src/main/scala/groksbeard/core/Onboarding.scala
package groksbeard.core

object Onboarding:
  val CliInstallHint =
    "Install the Grok Build CLI: curl -fsSL https://x.ai/cli/install.sh | bash && grok login"

  def missingCliMessage(searched: List[String]): String =
    s"Grok CLI not found. $CliInstallHint\nLooked in: ${searched.mkString(", ")}"
end Onboarding
